Libraries/src/Amazon.Lambda.AspNetCoreServer/AbstractAspNetCoreFunction.cs

```diff
@@ -251,6 +255,46 @@ protected virtual IHostBuilder CreateHostBuilder()
             return builder;
         }
 
+        #if NET8_0_OR_GREATER
+        /// <summary>
+        /// Return one or more <see cref="HttpRequestMessage"/>s that will be used to invoke
+        /// Routes in your lambda function in order to initialize the asp.net and lambda pipelines
+        /// during <see cref="SnapshotRestore.RegisterBeforeSnapshot"/>,
+        /// improving the performance gains offered by SnapStart.
+        /// <para />
+        /// The returned <see cref="HttpRequestMessage"/>s must have a relative
+        /// <see cref="HttpRequestMessage.RequestUri"/>.
+        /// <para />.
+        /// Be aware that this will invoke your applications function handler code
```

Comment:
Give the reason that function will be called multiple times which is to influence .NET JIT that this code is a hot path and should be optimized.

Reply:
added additional context